.gitignore

```diff
@@ -1,3 +1,5 @@
 node_modules
 dist
 RDKitStructureRenderer-*.tgz
+pkg
+public/RDKit_minimal.*
```

Comment:
`pkg` and the RDKit_minimal files are git ignored since they are managed by the build process during `npm run build` now